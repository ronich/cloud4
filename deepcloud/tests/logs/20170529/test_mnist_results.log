epoch,acc,loss,val_acc,val_loss
0,0.369416666667,2.06398067493,0.6621,1.11984451885
1,0.535083333302,1.44455857677,0.7388,0.893342353058
2,0.576800000032,1.35727380466,0.7882,0.954038900375
3,0.595266666698,1.3196902792,0.7624,0.88578637085
4,0.606633333349,1.30575523866,0.8119,0.855874331856
5,0.609100000032,1.29916287251,0.7385,0.946520631599
6,0.617966666667,1.32315967719,0.8069,1.01266576405
7,0.622499999968,1.34408749008,0.811,0.886112302208
8,0.628816666667,1.32166660786,0.7589,0.961145237064
9,0.631133333302,1.36044790923,0.7949,1.05305666714
10,0.637883333365,1.37000395387,0.844,0.955940988159
11,0.637783333365,1.37824760675,0.8004,0.93954669838
