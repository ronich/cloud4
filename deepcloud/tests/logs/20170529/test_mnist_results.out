Train on 60000 samples, validate on 10000 samples
Epoch 1/12
368s - loss: 2.0640 - acc: 0.3694 - val_loss: 1.1198 - val_acc: 0.6621
Epoch 2/12
358s - loss: 1.4446 - acc: 0.5351 - val_loss: 0.8933 - val_acc: 0.7388
Epoch 3/12
699s - loss: 1.3573 - acc: 0.5768 - val_loss: 0.9540 - val_acc: 0.7882
Epoch 4/12
3766s - loss: 1.3197 - acc: 0.5953 - val_loss: 0.8858 - val_acc: 0.7624
Epoch 5/12
3710s - loss: 1.3058 - acc: 0.6066 - val_loss: 0.8559 - val_acc: 0.8119
Epoch 6/12
3789s - loss: 1.2992 - acc: 0.6091 - val_loss: 0.9465 - val_acc: 0.7385
Epoch 7/12
3704s - loss: 1.3232 - acc: 0.6180 - val_loss: 1.0127 - val_acc: 0.8069
Epoch 8/12
3785s - loss: 1.3441 - acc: 0.6225 - val_loss: 0.8861 - val_acc: 0.8110
Epoch 9/12
3708s - loss: 1.3217 - acc: 0.6288 - val_loss: 0.9611 - val_acc: 0.7589
Epoch 10/12
3829s - loss: 1.3604 - acc: 0.6311 - val_loss: 1.0531 - val_acc: 0.7949
Epoch 11/12
3709s - loss: 1.3700 - acc: 0.6379 - val_loss: 0.9559 - val_acc: 0.8440
Epoch 12/12
3822s - loss: 1.3782 - acc: 0.6378 - val_loss: 0.9395 - val_acc: 0.8004
